.clangd

```diff
@@ -0,0 +1,2 @@
+CompileFlags:                     # Tweak the parse settings
+  Add: [-D__cpp_lib_coroutine=201902L]             # treat all files as C++, enable more warnings
```

Comment:
Why is this in here? This sounds like it'll interfere with a `$HOME/.clangd` file.

Reply:
It was committed accidentally, but you shouldn't have a `$HOME/.clangd` file anyways since that is reserved for project-specific settings: https://clangd.llvm.org/config#files

Reply:
`s#$HOME/.clangd#$HOME#.config/clangd/config.yaml` then, but same deal. If anything this should go into a separate PR, but defining something that interferes with the standard library sounds dangerous.